packages/cli/src/util/__tests__/colorizer.spec.ts

```diff
@@ -0,0 +1,98 @@
+import chalk from 'chalk';
+import {
+  PRE_PARAM_VALUE_TAG,
+  POST_PARAM_VALUE_TAG,
+  attachTagsToParamsValues,
+  transformPathParamsValues
+} from '../colorizer';
+
+describe('colorizer', () => {
+
+  describe('transformPathParamsValues()', () => {
+    it('colorizes tagged values of query params', () => {
+      const paramVal = 'sold,pending';
+      const path = `/no_auth/pets/findByStatus?status=${PRE_PARAM_VALUE_TAG}${paramVal}${POST_PARAM_VALUE_TAG}`;
+
+      expect(transformPathParamsValues(path, chalk.bold.blue)).toBe(`/no_auth/pets/findByStatus?status=${chalk.bold.blue(paramVal)}`);
+    });
+
+    it('colorizes tagged values of path params', () => {
+      const paramVal = 651;
+      const path = `/no_auth/pets/${PRE_PARAM_VALUE_TAG}${paramVal}${POST_PARAM_VALUE_TAG}`;
+
+      expect(transformPathParamsValues(path, chalk.bold.blue)).toBe(`/no_auth/pets/${chalk.bold.blue(`${paramVal}`)}`);
+    });
+
+    it('colorizes values that are equal to multiple or single , characters', () => {
```

Comment:
We don't need that test anymore since you have changed the way of handling param arrays.